packages/appium/lib/cli/extension-command.js

```diff
@@ -56,6 +57,26 @@ class ExtensionCommand {
     return this.config.extensionType;
   }
 
+  /**
+   * Logs a message and returns an {@linkcode Error} to throw.
+   *
+   * For TS to understand that a function throws an exception, it must actually throw an exception;
+   * in other words, _calling_ a function which always throws an exception is not enough.
+   * @param {string|Error} msgOrError
+   * @protected
+   * @returns {Error}
+   */
+  _fatalError(msgOrError) {
+    if (_.isString(msgOrError)) {
+      return new Error(this.log.decorate(msgOrError, 'error'));
+    }
+    if (_.isString(msgOrError.message)) {
```

Comment:
would this be enough to assume `msgOrError ` is of Error type or we have to check that explicitly?

Reply:
I think it's probably fair to restrict it to `string`, since I'm not actually passing an error elsewhere. We can add that later if needed.